conans/cli/output.py

```diff
@@ -0,0 +1,221 @@
+        colorama.init(convert=False, strip=False)
+        return True
+
+    isatty = hasattr(sys.stdout, "isatty") and sys.stdout.isatty()
+
+    clicolor = get_env("CLICOLOR")
+    if clicolor is not None:
+        if clicolor == "0" or not isatty:
+            return False
+        import colorama
+        colorama.init()
+        return True
+
+    # Respect color env setting or check tty if unset
+    color_set = "CONAN_COLOR_DISPLAY" in os.environ
+    if ((color_set and get_env("CONAN_COLOR_DISPLAY", 1))
+        or (not color_set and isatty)):
+        import colorama
+        if get_env("PYCHARM_HOSTED"):  # in PyCharm disable convert/strip
+            colorama.init(convert=False, strip=False)
+        else:
+            colorama.init()
+        color = True
+    else:
+        color = False
+    return color
+
+
+class Color(object):
+    """ Wrapper around colorama colors that are undefined in importing
+    """
+    RED = Fore.RED  # @UndefinedVariable
+    WHITE = Fore.WHITE  # @UndefinedVariable
+    CYAN = Fore.CYAN  # @UndefinedVariable
+    GREEN = Fore.GREEN  # @UndefinedVariable
+    MAGENTA = Fore.MAGENTA  # @UndefinedVariable
+    BLUE = Fore.BLUE  # @UndefinedVariable
+    YELLOW = Fore.YELLOW  # @UndefinedVariable
+    BLACK = Fore.BLACK  # @UndefinedVariable
+
+    BRIGHT_RED = Style.BRIGHT + Fore.RED  # @UndefinedVariable
+    BRIGHT_BLUE = Style.BRIGHT + Fore.BLUE  # @UndefinedVariable
+    BRIGHT_YELLOW = Style.BRIGHT + Fore.YELLOW  # @UndefinedVariable
+    BRIGHT_GREEN = Style.BRIGHT + Fore.GREEN  # @UndefinedVariable
+    BRIGHT_CYAN = Style.BRIGHT + Fore.CYAN  # @UndefinedVariable
+    BRIGHT_WHITE = Style.BRIGHT + Fore.WHITE  # @UndefinedVariable
+    BRIGHT_MAGENTA = Style.BRIGHT + Fore.MAGENTA  # @UndefinedVariable
+
+
+if get_env("CONAN_COLOR_DARK", 0):
+    Color.WHITE = Fore.BLACK
+    Color.CYAN = Fore.BLUE
+    Color.YELLOW = Fore.MAGENTA
+    Color.BRIGHT_WHITE = Fore.BLACK
+    Color.BRIGHT_CYAN = Fore.BLUE
+    Color.BRIGHT_YELLOW = Fore.MAGENTA
+    Color.BRIGHT_GREEN = Fore.GREEN
+
+try:
+    from logging import NullHandler
+except ImportError:
+    class NullHandler(logging.Handler):
+        def handle(self, record):
+            pass
+
+        def emit(self, record):
+            pass
+
+        def createLock(self):
+            self.lock = None
+
+
+class TqdmHandler(logging.StreamHandler):
+    def __init__(self, stream=None):
+        self._stream = stream
+        super(TqdmHandler, self).__init__(stream)
+
+    def emit(self, record):
+        try:
+            msg = self.format(record)
+            tqdm.tqdm.write(msg, file=self._stream)
+            self.flush()
+        except (KeyboardInterrupt, SystemExit):
+            raise
+        except:
+            self.handleError(record)
+
+
+class ConanOutput(object):
+    def __init__(self, stream=None, color=False):
+        self._logger_name = "conan.output"
+        self._stream = stream
+        self._stream_handler = None
+
```

Comment:
I would store the log as a member variable, is there a reason not to do it?

```python
log = logging.getLogger("conan.output")
```

...and probably there is no need to store the _logger_anme